guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -24,13 +19,23 @@ ifndef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --disable '*'
+# subscription-manager repos \
+ifdef::satellite[]
+--enable={RepoRHEL8ServerSatelliteServerProductVersion} \
+endif::[]
```

Comment:
There is no equivalent instruction for enabling the katello repos for upstream. Something like this should be close:
```
ifdef::katello[]
:distribution: rhel
:distribution-major-version: 8
:package-manager: dnf
include::proc_configuring-repositories-el.adoc[]
endif::[]
```

The only downside is that it also installs puppet which is actually not needed here.